solidjs-tailwind/src/components/UserProfileCard/user-profile-card.classNames.js

```diff
@@ -0,0 +1,12 @@
+export const avatar = 'rounded-full shadow z-30';
+export const name = 'text-2xl text-gray-800 font-bold leading-tight';
```

Comment:
Since Solidjs support CSS module, I think we should change the styles on this PR, same way it was done in Nextjs showcase.

Remix used this pattern because it does not support CSS module

Reply:
This will be merged and a ticket will be created to fix all styling issues